lib/R.sc

```diff
@@ -1277,36 +1283,50 @@ RSineOscillatorModule : RModule {
 	*params {
 		^[
 			'Range' -> (
-				Spec: ControlSpec.new(-2, 2, 'lin', 1, 0),
+				Spec: ControlSpec.new(-8, 8, nil, 1, 0),
 				LagTime: 0.01
 			),
 			'Tune' -> (
-				Spec: ControlSpec.new(-600, 600, 'lin', 0, 0, "cents"),
+				Spec: ControlSpec.new(-600, 600, nil, 0, 0, "cents"),
 				LagTime: 0.01
 			),
 			'FM' -> (
 				Spec: \unipolar.asSpec, // TODO: bipolar?
 				LagTime: 0.01
 			),
+			'LinFM' -> (
+				Spec: ControlSpec.new(0, 5, 'lin', 0.01, 0, ""),
+				LagTime: 0.2
+			),
+			'PM' -> (
+				Spec: ControlSpec.new(0, 5, 'lin', 0.01, 0, ""),
+				LagTime: 0.2
+			),
```

Comment:
Not sure about the range: Would 0-1 (presented as 0-100%) be more clear?

Reply:
see comment above - I used this range because of controlspec weirdness but I can probably revisit and use 0-200% and keep the same feel. (100% = "full brightness", 100%+ = "too bright")